providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/FlagdOptions.java

```diff
@@ -92,6 +92,14 @@ public class FlagdOptions {
     @Builder.Default
     private int deadline = fallBackToEnvOrDefault(Config.DEADLINE_MS_ENV_VAR_NAME, Config.DEFAULT_DEADLINE);
 
+    /**
+     * Streaming connection deadline in milliseconds.
+     * Set to 0 to disable the deadline.
```

Comment:
```suggestion
     * Streaming connection deadline in milliseconds.
     * Set to 0 to disable the deadline.
     * Defaults to 600000 (10 minutes); recommended to prevent infrastructure from killing idle connections.
```